### PR TITLE
[Communication] phone number code extraction

### PR DESCRIPTION
**Pull request by sacheu**

### CODE REVIEW

**Comment by sacheu** [comment]
/axp run python - communication - ci

File: sdk/communication/azure-communication-phonenumber/test/test_phone_number_administration_client.py [hunk 1]
```diff
@@ -17,9 +17,9 @@
 from phone_number_testcase import PhoneNumberCommunicationTestCase
 from _shared.testcase import BodyReplacerProcessor
 from _shared.utils import create_token_credential
-from azure.communication.administration._shared.utils import parse_connection_str
+from azure.communication.phonenumber._shared.utils import parse_connection_str
 
-SKIP_PHONE_NUMBER_TESTS = True
+SKIP_PHONE_NUMBER_TESTS = False
```
Comment (jbeauregardb) [inline sdk/communication/azure-communication-phonenumber/test/test_phone_number_administration_client.py:22]: Let's revert this back to True

File: sdk/communication/azure-communication-phonenumbers/sdk_packaging.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[packaging]
+auto_update = false
+package_name = "azure-communication-phonenumbers"
+package_pprint_name = "Communication Phone Number"
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/sdk_packaging.toml:4]: Communication Phone Numbers

Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/sdk_packaging.toml:4]: Communication Phone Numbers

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=master)
 
-# Azure Communication Administration Package client library for Python
+# Azure Communication  Package client library for Python
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/README.md:3]: nit: There is an extra space after Communication

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
-Azure Communication Administration client package is used to administer Phone Numbers. 
+Azure Communication Phone Number client package is used to administer Phone Numbers.
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/README.md:5]: Phone Numbers

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
+Azure Communication Phone Number client package is used to administer Phone Numbers. 
 
 # Getting started
 ### Prerequisites
 - Python 2.7, or 3.5 or later is required to use this package.
 - You must have an [Azure subscription](https://azure.microsoft.com/free/)
 - A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
 ### Install the package
-Install the Azure Communication Administration client library for Python with [pip](https://pypi.org/project/pip/):
+Install the Azure Communication Phone Number client library for Python with [pip](https://pypi.org/project/pip/):
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/README.md:13]: Phone Numbers

File: sdk/communication/azure-communication-phonenumbers/README.md [hunk 6]
```diff
@@ -172,7 +172,7 @@ The Phone Number Administration client will raise exceptions defined in [Azure C
 # Next steps
 ## More sample code
 
-Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/communication/azure-communication-administration/samples) directory for detailed examples of how to use this library.
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/communication/azure-communication-phonenumber/samples) directory for detailed examples of how to use this library.
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-phonenumbers/README.md:175]: `phonenumbers`

**Review by RezaJooyandeh — APPROVED** [review]
LGTM with a couple of small comments. Thank you 🙏🏻

**Comment by annatisch** [comment]
Hi @sacheu - has this namespace/package change been through archboard approval?
If so - could you point me to the archboard meeting or email thread?

**Comment by sacheu** [comment]
> Hi @sacheu - has this namespace/package change been through archboard approval?
> If so - could you point me to the archboard meeting or email thread?

Yes. this change has been thru board review @RezaJooyandeh  do you have the link?

**Pull request merged**